Mars-SeleniumAutomation/Pages/Components/ProfilePageOverview/ProfileMenuTab.cs

```diff
@@ -0,0 +1,147 @@
+﻿using Mars_SeleniumAutomation.Utilities;//*[@id="account-profile-section"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/tbody/tr/td[3]/span[2]/i
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using static System.Collections.Specialized.BitVector32;
+using TechTalk.SpecFlow.Assist;
+
+namespace Mars_SeleniumAutomation.Pages.Components.ProfilePageOverview
+{
+    public class ProfileMenuTab : Base
+    {
+#pragma warning disable
+        private IWebElement userNameDropdownIcon;
+        private IWebElement availabilityEditBtn;
+        private IWebElement hoursEditBtn;
+        private IWebElement earnTargetEditBtn;
+        private IWebElement descriptionEditIcon;
+        private IWebElement languagesTab;
+        private IWebElement languageEditIcon;
+        private IWebElement languageDeleteIcon;
+        private IWebElement shareSkillTab;
+        private IWebElement manageListingTab;
+        private IWebElement manageListing;
+        private IWebElement manageListingEditIcon;
+        public void renderComponents()
+        {
+            try
+            {
+                userNameDropdownIcon = driver.FindElement(By.XPath("//div[@class='title']//i[@class='dropdown icon']"));
+                availabilityEditBtn  = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]//section[2]//div[2]//div[3]//div[2]/div/span/i"));
+                hoursEditBtn         = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]//section[2]//div[2]//div/div/div[3]//div[3]/div/span/i"));
+                earnTargetEditBtn    =  driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]//section[2]//div[2]//div[3]//div[4]/div/span/i"));
+                descriptionEditIcon  = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/div/div/div/h3/span/i"));
+            }
+            catch(Exception ex) 
+            {
+                Console.WriteLine(ex);
+            }
+        }
+        public void renderComponents1()
```

Comment:
can you rename this method to something meaningful please, what component does this render